wiki/en/Running-a-Server.md

```diff
@@ -18,50 +18,39 @@ permalink: "/wiki/Running-a-Server"
 </details>
 
 
-
 ## Do I need to run a Server?
 
-The short answer is **no**. For various reasons, you should try to use Servers listed by the built in Directories, at least at first while getting your sound set up with the Jamulus Client. You may also be able to use a Server hosted by a third party. Doing so will save you the trouble of setting one up yourself.  
-
-Note that you can have a "private" session with other people on a Server by simply soloing each other. You will then not be able to hear anyone else if they enter your Server (note that this does not stop them from hearing you, using the chat function, or seeing your profile information).
+**No**. You can use the Servers listed by the built in Directories and use Jamulus without running a server.
 
```

Comment:
It probably warrants another section heading at this point (i.e. after line 23 (new)), given the new approach.

Reply:
See my new commit.